src/plugins/score-plugin-avnd/Crousti/Painter.hpp

```diff
@@ -151,6 +199,120 @@ struct QPainterAdapter
     {
         path.addEllipse(QPointF{cx, cy}, cr, cr);
     }
+
+    // Antoine
+    //          tab, count
+    void draw_polygon(double* tab, int count)
+    {
+        QPolygonF poly;
+        double x, y;
+        for (int i = 0; i < count*2; i+=2){
+            x = tab[i];
+            y = tab[i+1];
+            poly << QPointF(x, y);
+        }
+        path.addPolygon(poly);
+    }
+
+    // Antoine
+    //          x, y, radius1, radius2, count
+    void draw_star(float x, float y, float r1, float r2, int count)
```

Comment:
This function should be expressed as a separate algorithm which uses this class I think